omero/sysadmins/mail.txt

```diff
@@ -0,0 +1,88 @@
+OMERO.mail
+==========
+
+The OMERO server has the ability to send email to any users who have a properly
+configured email address. OMERO system administrators can then use the
+:omerocmd:`admin email` command to contact those users.
+
+In order to activate the subsystem, minimally the :property:`omero.mail.config`
+property will need to be activated.
+
+Minimum configuration
+---------------------
+
+::
+
+    $ bin/omero config set omero.mail.config true
+
+By default, this will use "localhost" as the mail server on port 25 and send as
```

Comment:
Looks like single back ticks are what's used elsewhere, there is no specific role for config values only commands and environment variables.